common/src/BasePlugin.php

```diff
@@ -2,6 +2,7 @@
 namespace Valued\WordPress;
 
 use ReflectionClass;
+use \Automattic\WooCommerce\Utilities\FeaturesUtil;
```

Comment:
`use` declarations don't need a leading `\`.

```suggestion
use Automattic\WooCommerce\Utilities\FeaturesUtil;
```